packages/profiling-node/test/spanProfileUtils.test.ts

```diff
@@ -322,3 +360,191 @@ describe('spanProfileUtils', () => {
     });
   });
 });
+
+describe('continuous profiling', () => {
+  beforeEach(() => {
+    jest.useFakeTimers();
+    // We will mock the carrier as if it has been initialized by the SDK, else everything is short circuited
+    getMainCarrier().__SENTRY__ = {};
+    GLOBAL_OBJ._sentryDebugIds = undefined as any;
+  });
+  afterEach(() => {
+    const client = Sentry.getClient();
+    const integration = client?.getIntegrationByName('ProfilingIntegration');
```

Comment:
can we import and use the type here?

```suggestion
    const integration = client?.getIntegrationByName<ProfilingIntegration>('ProfilingIntegration');
```